### PR TITLE
Pull Request: Logic Complete and Ready to Deploy (Milestone 5)

### PR DESCRIPTION
**Pull request by johnsonsirv**
**Request to Merge <===** [Live Link](https://mvp-react-calculator.herokuapp.com/)

**In this PR I achieved the following:** :rocket: 

#### PS: This is a Solo Project

***Thanks in advance. Happy Reviewing!!*** :sunglasses:

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -2,29 +2,117 @@ import React, { Component } from 'react';
...
-    const { result } = this.state;
+    let { result, statusText } = this.state;
+    const { finished, finishedTotal } = this.state;
+    result = finished ? finishedTotal : result;
+    statusText = finished ? 'Ready' : statusText;
     return (
       <div className="app">
-        <Display result={result} />
+        <Display result={result} statusText={statusText} />
         <ButtonPanel clickHandler={this.handleClick} />
```
Comment (shloch) [inline src/components/App.js]: it should pass TOTAL or NEXT to DIsplay component
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/passToatalToDisplay.png

**Review by shloch — CHANGES_REQUESTED** [review]
hello @johnsonsirv 
Everything works perfectly bro, i have just one requirement

“if i want to do “5-3”, i get a “2" which is fine. But after if i want to continue by dailing [multiplication symbol] , then 6 so that i get a 12 .... that is not possible becuse your calculator re-initialises to zero after every operation. The output of ONE operation should be able to be used as INPUT into another operation. There’s a reason the AC key is there
Please debug this

Thanks, waiting for last re-review hopefully 
Happy coding

File: src/components/App.js (hunk 1 above)
Comment (johnsonsirv) [reply src/components/App.js]: Yes. It passes total or next accordingly to the ```Display``` but with a variable name called ``result``
I am thiniking that requirement is talking about the value of _total_ or _next_

**Review by jofer86 — APPROVED** [review]
## 👋 👋 Hello @johnsonsirv !! I can see you tackled the changes required by @shloch ! Well done! This PR is now Approved you might merge!! 🚀 🚀

**Comment by johnsonsirv** [comment]
Thank you @jofer86 and @shloch for your reviews.

**Pull request merged**